<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" multifile-model="true">
    <data-source source="LOCAL" name="Django default" uuid="1382864e-1b41-4f56-a9d2-f3de48cef704">
      <driver-ref>mysql</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>$PROJECT_DIR$/ftpsidtm/settings.py</remarks>
      <jdbc-driver>com.mysql.jdbc.Driver</jdbc-driver>
      <jdbc-url>jdbc:mysql://mysql.cafeperfeito.com.br:3306/cafeperfeito</jdbc-url>
      <driver-properties>
        <property name="autoReconnect" value="true" />
        <property name="yearIsDateType" value="false" />
        <property name="characterEncoding" value="utf8" />
        <property name="zeroDateTimeBehavior" value="convertToNull" />
        <property name="characterSetResults" value="utf8" />
        <property name="tinyInt1isBit" value="false" />
      </driver-properties>
    </data-source>
  </component>
</project>